tests/brayns.cpp

```diff
@@ -120,3 +121,21 @@ BOOST_AUTO_TEST_CASE(defaults)
     BOOST_CHECK_EQUAL(scene.getBounds(), defaultBoundingBox);
     BOOST_CHECK(geomParams.getMemoryMode() == brayns::MemoryMode::shared);
 }
+
+BOOST_AUTO_TEST_CASE(bvh_type)
+{
+    auto& testSuite = boost::unit_test::framework::master_test_suite();
+    const char* app = testSuite.argv[0];
+    const char* argv[] = {
+        app,       "demo", "--default-bvh-flag", "robust", "--default-bvh-flag",
+        "compact",
```

Comment:
I would have never guessed this is how you provide multiple flags through the command line...